### PR TITLE
[PAGE] Create initial Cyborg Guide page

### PR DESCRIPTION
**Pull request by lkinas**
Create initial Cyborg Guide page [based on this wireframe.](https://www.figma.com/file/mM2SeJGbDQN65FQ78JAIAO/KSM-Society-Wireframing?node-id=617%3A34758)

# Concerns
- the vertical line is not well aligned and responsive (help here would be appreciated <3 )
- haven't implemented a scroll animation through levels

### CODE REVIEW

File: src/pages/CyborgGuide.tsx [hunk 1]
```diff
@@ -1,9 +1,267 @@
...
+                rel="noreferrer"
+              >
+                Kusama society guide
+              </a>
+            </p>
+            <p>
+              <a
+                target="_blank"
+                // eslint-disable-next-line max-len
+                href="https://polkascan.io/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17"
```
Comment (laurogripa) [inline src/pages/CyborgGuide.tsx:42]: Let's extract this URL to a constant like SOCIETY_RULES_URL

File: src/pages/CyborgGuide.tsx [hunk 2]
```diff
@@ -1,9 +1,267 @@
...
+      <Container>
+        <GuideRow>
+          <Col>
+            <div className="h1 text-uppercase font-weight-bold">
+              The journey
+            </div>
+            <p className="mb-1">
+              To become a Cyborg means to get a membership in the Kappa Sigma Mu
+              Society.
+            </p>
```
Comment (laurogripa) [inline src/pages/CyborgGuide.tsx:60]: I think we should start thinking about i18n

File: src/pages/CyborgGuide.tsx [hunk 3]
```diff
@@ -1,9 +1,267 @@
...
+            </strong>
+            <p className="mb-1">
+              When your bid gets accepted, you become a Candidate. Now it’s time
+              to get tattoed!
+            </p>
+            <p>
+              <a
+                target="_blank"
+                // eslint-disable-next-line max-len
+                href="https://polkascan.io/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17"
```
Comment (laurogripa) [inline src/pages/CyborgGuide.tsx:162]: Why this repeats? WDYT of extracting it to a simple reusable component?

File: src/pages/CyborgGuide.tsx (hunk 2 above)
Comment (alfakini) [reply src/pages/CyborgGuide.tsx:75]: Added a card to Trello.

**Review by alfakini — APPROVED** [review]

**Comment by alfakini** [comment]
Merging this PR. Added a card to the Trello board to address the issues: https://trello.com/c/wtwbrGDV/216-page-improve-cyborg-journey-page

**Pull request merged**